src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -1174,9 +1175,14 @@ private void fetchOrCreateLocationPIO(ExperimentObservation importRow) {
     }
 
     private PendingImportObject<BrAPITrial> fetchOrCreateTrialPIO(Program program, User user, boolean commit, ExperimentObservation importRow, Supplier<BigInteger> expNextVal) throws UnprocessableEntityException {
-        PendingImportObject<BrAPITrial> pio;
+        PendingImportObject<BrAPITrial> trialPio;
         if (trialByNameNoScope.containsKey(importRow.getExpTitle())) {
-            pio = trialByNameNoScope.get(importRow.getExpTitle());
+            PendingImportObject<BrAPIStudy> envPio;
+            trialPio = trialByNameNoScope.get(importRow.getExpTitle());
+            envPio = this.studyByNameNoScope.get(importRow.getEnv());
+            if  (trialPio.getState() == ImportObjectState.EXISTING && (StringUtils.isBlank( importRow.getObsUnitID() )) && (envPio!=null && ImportObjectState.EXISTING==envPio.getState() ) ){
+                throw new UnprocessableEntityException(PREEXISTING_EXPERIMENT_TITLE);
```

Comment:
A possible NPE if line 1181 returns null can be avoided 
```suggestion
            if  (trialPio != null && ImportObjectState.EXISTING == trialPio.getState() && (StringUtils.isBlank( importRow.getObsUnitID() )) && (envPio!=null && ImportObjectState.EXISTING==envPio.getState() ) ){
                throw new UnprocessableEntityException(PREEXISTING_EXPERIMENT_TITLE);
```